mne/stats/regression.py

```diff
@@ -246,13 +246,12 @@ def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
     """
 
     if isinstance(solver, string_types):
-        if solver == 'pinv':
-            fast_dot = _get_fast_dot()
+        if solver == 'cholesky':
+            def solver(X, y):
+                a = (X.T * X).toarray()
```

Comment:
Tried that, it didn't work. I think sparse matrices don't have the same methods as arrays (something about the matrix object being generally weird). So we could first convert it, but I think it's much more efficient to first do the dot product and then covert?